qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/web/views/create/offer/CustomerSelectionView.groovy

```diff
@@ -64,8 +64,6 @@ class CustomerSelectionView extends VerticalLayout{
      * Initializes the start layout of this view class
      */
     private void initLayout(){
-        Label titleLabel = new Label("Select Customer")
```

Comment:
Why was this removed, I guess because it overlapped with the navbar? 
I think it makes sense to have a title for each step of offer generation.  
A possible solution could be to set a spacing layout between navbar and label so this doesn't happen

Reply:
No, I think it just requires space in the view and is redundant since the bar indicates that already